vector/src/main/java/im/vector/app/features/home/HomeDetailFragment.kt

```diff
@@ -352,8 +357,12 @@ class HomeDetailFragment @Inject constructor(
             if (fragmentToShow == null) {
                 when (tab) {
                     is HomeTab.RoomList -> {
-                        val params = RoomListParams(tab.displayMode)
-                        add(R.id.roomListContainer, RoomListFragment::class.java, params.toMvRxBundle(), fragmentTag)
+                        if (vectorFeatures.isNewAppLayoutEnabled()) {
+                            add(R.id.roomListContainer, createHomeRoomListFragment(), fragmentTag)
```

Comment:
Is it necessary to invoke the `fragmentFactory` here? Just wondering if we could just to 

```suggestion
                            add(R.id.roomListContainer, HomeRoomListFragment::class.java, null, fragmentTag)
```

Reply:
I thought to follow existing pattern, taking in account that there could be params for HomeRoomListFragment later. But I think it's better to simplify for now and then extract to method if it will be necessary